File: Razor/Models/Pays.cs

namespace Models;

using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;


[Table("pays")]
public class Pays {
    /// Field
    [Key]
    [Column("id_pays")]
    public int IdPays { get; set; }
    
    [Column("pays_name")]
    public string PaysName { get; set; }
    
    
    
    /// Constructor
    public Pays(){}
    
    
    /// Getter and setter
    
}

File: Thymeleaf/com/bezkoder/spring/thymeleaf/entity/Pays.java

package com.bezkoder.spring.thymeleaf.entity;

import jakarta.persistence.*;


@Entity
@Table(name = "pays")
public class Pays {
    /// Field
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id_pays")
    Integer idPays;
    
    @Column(name = "pays_name")
    String paysName;
    
    
    
    /// Constructor
    public Pays(){}
    
    
    /// Getter and setter
    public Integer getIdPays() {
        return idPays;
    }
    
    public void setIdPays(Integer idPays) {
        this.idPays = idPays;
    }
    
    public String getPaysName() {
        return paysName;
    }
    
    public void setPaysName(String paysName) {
        this.paysName = paysName;
    }
    
    
}
